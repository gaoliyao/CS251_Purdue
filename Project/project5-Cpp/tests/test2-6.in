2
3 3
IND ORD 163.30
ORD DTW 238.30
IND DTW 239.90
IND ORD
DTW IND
DTW ORD
END
