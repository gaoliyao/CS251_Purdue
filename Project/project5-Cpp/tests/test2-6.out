IND ORD 163.30
DTW IND 239.90
DTW ORD 238.30
